aidants_connect_web/forms.py

```diff
@@ -126,18 +130,45 @@ def clean(self):
 class MandatForm(forms.Form):
     DEMARCHES = [(key, value) for key, value in settings.DEMARCHES.items()]
     demarche = forms.MultipleChoiceField(
-        choices=DEMARCHES, required=True, widget=forms.CheckboxSelectMultiple
+        choices=DEMARCHES,
+        required=True,
+        widget=forms.CheckboxSelectMultiple,
+        error_messages={"required": _("Vous devez sélectionner au moins une démarche")},
     )
 
     # models.MandatDureeKeywords
     DUREES = [
         ("SHORT", {"title": "Mandat court", "description": "(expire demain)"}),
         ("LONG", {"title": "Mandat long", "description": "(12 mois)"}),
     ]
-    duree = forms.ChoiceField(choices=DUREES, required=True, initial=3)
+    duree = forms.ChoiceField(
+        choices=DUREES,
+        required=True,
+        initial=3,
+        error_messages={"required": _("Veuillez sélectionner la durée du mandat")},
```

Comment:
Messages d'erreurs plus spécifiques. Voir [r612566098](https://github.com/betagouv/Aidants_Connect/pull/343/files#r612566098).

Reply:
On mettrait pas des points à la fin des messages ?
```suggestion
        error_messages={"required": _("Veuillez sélectionner la durée du mandat.")},
```